### PR TITLE
feature/interlok-3509 json lines

### PR DESCRIPTION
**Pull request by albinoloverats**
## Motivation

JdbcJsonOutput should have a JSON_LINES equivalent completeness there needs to be a JdbcJsonLinesOutput that mirrors what JdbcJsonOutput does. JsonBlobListRenderer should have a JSON lines variant. There should be a bloblist renderer variant that emits JSON_Lines.

## Modification

Add two new classes: `JdbcJsonOutputLines` and `JsonBlobListRendererLines` that

## Result

The JSON is output with each object on its own line.

## Testing

Confirm that the JSON output is as expected.

### CODE REVIEW

**Comment by quotidian-ennui** [comment]
JSON_LINES isn't an array, so what you've done isn't in spec...  It's essentially a new line terminated list of JSON objects. No pretty print, nothing. Essentially each JSON object lives on a new line...

```
{"key1":"value1", "key2":"value", "key3":"value".... (so on and so forth maybe hundreds of thousands of values)}
{"key1:"value2"}
```

**Comment by albinoloverats** [comment]
Ah okay, so it was complete halfway through what I did, removing both `generator.writeStartArray();` and `generator.writeEndArray();`, before I erroneously wrapped them in [].

File: interlok-json/src/main/java/com/adaptris/core/json/jdbc/JdbcJsonOutputLines.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ *
+ * <p>
+ * Takes the first result set and writes out each row as part of a json
+ * array as a new line. You should use this output implementation if
+ * you are executing a SELECT via {@link JdbcDataQueryService}.
+ * </p>
+ *
+ * <pre>
+ {@code
+    [ { "firstName": "John", "lastName": "Doe" },
```
Comment (aaron-mcgrath-adp) [inline interlok-json/src/main/java/com/adaptris/core/json/jdbc/JdbcJsonOutputLines.java:36]: Should this be wrapped in braces?

Comment (albinoloverats) [reply interlok-json/src/main/java/com/adaptris/core/json/jdbc/JdbcJsonOutputLines.java:36]: Ah yes, I'll update the JavaDocs.

**Review by aaron-mcgrath-adp — APPROVED** [review]
Conceptually it looks fine.
Without a quick start testing guide I haven't created a database/data and tested the service output however.

**Pull request merged**